From ce1525716b247b0c8eec8a9f0adb2c4009dcdf23 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Tue, 29 Apr 2025 13:53:01 +0100
Subject: [PATCH] v2v: Remove --password-file option

This was changed to '-ip' in commit eb508ba22d ("v2v: Use -ip to pass
input password (instead of --password-file).", June 2018).  It also
now can be confused with the similar --password and --password-crypto
options, used by the virt-customize code.

(cherry picked from commit 21d914d6b2443d2f41ef62c7f185e188de4a1aab)
---
 docs/test-docs.sh | 2 --
 v2v/v2v.ml        | 2 --
 2 files changed, 4 deletions(-)

diff --git a/docs/test-docs.sh b/docs/test-docs.sh
index 1037bf7e..9a4c58ab 100755
--- a/docs/test-docs.sh
+++ b/docs/test-docs.sh
@@ -78,7 +78,6 @@ $srcdir/../podcheck.pl virt-v2v.pod virt-v2v \
 --ip,\
 --it,\
 --in-place,\
---password-file,\
 --oa,\
 --oc,\
 --of,\
@@ -96,7 +95,6 @@ $srcdir/../podcheck.pl virt-v2v-in-place.pod virt-v2v-in-place \
 --io,\
 --ip,\
 --it,\
---password-file,\
 --oa,\
 --oc,\
 --of,\
diff --git a/v2v/v2v.ml b/v2v/v2v.ml
index 30f317ee..7f1d4352 100644
--- a/v2v/v2v.ml
+++ b/v2v/v2v.ml
@@ -244,8 +244,6 @@ let rec main () =
                                     s_"Use password from file to connect to output hypervisor";
     [ M"os" ],       Getopt.String ("storage", set_string_option_once "-os" output_storage),
                                     s_"Set output storage location";
-    [ L"password-file" ], Getopt.String ("filename", set_string_option_once "-ip" input_password),
-      s_"Same as ‘-ip filename’";
     [ L"parallel" ], Getopt.Set_int ("N", parallel),
       s_"Run up to N instances of nbdcopy in parallel";
     [ L"print-source" ], Getopt.Set print_source,
